dev/dags/dbt/altered_jaffle_shop/profiles.yml

```diff
@@ -0,0 +1,24 @@
+default:
```

Comment:
Both profile looks very similar, shall we remove one?

Reply:
Yes, I've logged a follow-up ticket for this (#1384) since both our profiles share this problem, and we'll need to change the K8s example and docs.